settings.gradle.kts

```diff
@@ -126,92 +127,8 @@ dependencyResolutionManagement {
-            version("com.swirlds.virtualmap", swirldsVersion)
-            version("dagger", daggerVersion)
-            version("dagger.compiler", daggerVersion)
-            version("grpc.protobuf", ioGrpcVersion)
-            version("grpc.stub", ioGrpcVersion)
-            version("headlong", "6.1.1")
-            version("info.picocli", "4.6.3")
-            version("io.github.classgraph", "4.8.65")
-            version("io.grpc", helidonVersion)
-            version("io.helidon.grpc.client", helidonVersion)
-            version("io.helidon.grpc.core", helidonVersion)
-            version("io.helidon.grpc.server", helidonVersion)
-            version("io_helidon_common_configurable", helidonVersion)
-            version("java.annotation", "3.0.2")
-            version("javax.inject", "1")
-            version("net.i2p.crypto.eddsa", "0.3.0")
-            version("org.antlr.antlr4.runtime", "4.11.1")
-            version("org.apache.commons.codec", "1.15")
-            version("org.apache.commons.collections4", "4.4")
-            version("org.apache.commons.io", "2.11.0")
-            version("org.apache.commons.lang3", "3.12.0")
-            version("org.apache.logging.log4j", log4jVersion)
-            version("org.apache.logging.log4j.core", log4jVersion)
-            version("org.apache.logging.log4j.jul", log4jVersion)
-            version("org.assertj.core", "3.23.1")
-            version("org.bouncycastle.pkix", bouncycastleVersion)
-            version("org.bouncycastle.provider", bouncycastleVersion)
-            version("org.eclipse.collections.api", eclipseCollectionsVersion)
-            version("org.eclipse.collections.impl", eclipseCollectionsVersion)
-            version("org.hamcrest", "2.2")
-            version("org.hyperledger.besu.crypto", besuVersion)
-            version("org.hyperledger.besu.datatypes", besuVersion)
-            version("org.hyperledger.besu.evm", besuVersion)
-            version("org.hyperledger.besu.secp256k1", besuNativeVersion)
-            version("org.json", "20210307")
-            version("org.junit.jupiter.api", "5.9.0")
-            version("org.junitpioneer", "2.0.1")
-            version("org.mockito", mockitoVersion)
-            version("org.mockito.inline", mockitoVersion)
-            version("org.mockito.junit.jupiter", mockitoVersion)
-            version("org.opentest4j", "1.2.0")
-            version("org.slf4j", "2.0.3")
-            version("org.testcontainers", testContainersVersion)
-            version("org.testcontainers.junit.jupiter", testContainersVersion)
-            version("tuweni.bytes", tuweniVersion)
-            version("tuweni.units", tuweniVersion)
-            version("uk.org.webcompere.systemstubs.core", systemStubsVersion)
-            version("uk.org.webcompere.systemstubs.jupiter", systemStubsVersion)
-
+            version("google-proto", "3.19.4")
+            version("grpc-proto", "1.45.1")
```

Comment:
Initially, I wanted to put these into the platform as well. But the 'protobuf' plugin is very stubborn in picking this up for the internal proto tool dependency resolution it does. I am sure I can find a way though if we absolutely want it.

However there are pro and cons, so I left it here for now to postpone this decision.

PRO (move to platform)
- everything in one place
- there probably is a relationship between protobuf modules (e.g. com.google.protobuf) and the protobuf compiler(s). So the version – 3.19.4 – is repeated.

CON (keep it here)
- The proto compilers are not Modules to they can't be addressed by module name. But it would be nice to use module names only in the platform. But it's not a must. (Or we patch the compilers only for the sake of having a name...)
- Mixing tool versions and library/module versions in one place. It's a bit of a philosophical question if this is "bad".

I think I would tend toward moving these into the platform as well eventually. But I would do that as a follow up then.

Reply:
I agree with a follow up PR on this and with moving to the platform (BOM). We should also look at updating helidon to at least version 3.1.0 as well.